api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -668,16 +703,16 @@ public CohortDefinition tbSuspect() {
 	 * Individuals tested more than twice in the last 12 months
 	 * @return CohortDefinition
 	 */			
-	public CohortDefinition individualsTestedMoreThanTwiceInLast12Months() {
```

Comment:
@solemabrothers Why delete the old definition which is used in previous reports? They are now broken. Just create a new definition and leave the old one untouched